keras/backend/tensorflow_backend.py

```diff
@@ -280,6 +280,31 @@ def to_dense(tensor):
 name_scope = tf.name_scope
 
 
+def is_placeholder(tensor):
+    """Returns whether a tensor is a placeholder.
+
+    # Arguments
+        tensor: A tensor instance.
+
+    # Returns
+        A boolean.
+
+    # Example
+    ```python
+        >>> from keras import backend as K
+        >>> a = K.placeholder((2, 2), sparse=False)
+        >>> print(K.is_placeholder(a))
+        True
+    ```
+    """
+    try:
+        if isinstance(tensor, tf.Variable):
+            return True
+        return tensor._is_placeholder
+    except AttributeError:
+        return True
```

Comment:
Yeah but most (if not all) layers are not placeholder, right? Beside Input, I don't know any layer which could be placeholder.